homework-submission/todoapp/src/Components/TodoElement.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { observer, inject } from 'mobx-react';
+
+@inject('todos')
+@observer
+class TodoItem extends React.Component {
+  handleUpdate = (e) => {
+    e.preventDefault()
+    const todo = this.props.todo
+    const { updateHandler, editHandler } = this.props.todos
+    let oldTask = todo.description
+    let newTask = this.editField.value
+    let newDate = this.secondEditField.value
+    updateHandler(oldTask, newTask, newDate)
+    editHandler(todo.id)
+  }
+  render() {
+    const todo = this.props.todo;
+    const { deleteHandler, editHandler, checkBoxHandler } = this.props.todos;
+    let TodoView;
+    if (!todo.edit) {
+      TodoView = (
+        <div className="todo">
+          <input type='checkbox' id='description' defaultChecked={todo.done} onChange={() => checkBoxHandler(todo.id)} />
```

Comment:
We may rephrase this a bit to be `checked` to solve the following:

```suggestion
          <input type='checkbox' id='description' checked={todo.done} onChange={() => checkBoxHandler(todo.id)} />
```

![ezgif com-video-to-gif 2](https://user-images.githubusercontent.com/25272822/49102771-f9c1ea80-f279-11e8-8fd9-c791fb9a16bf.gif)